changelogs/fragments/4557-alternatives-add-activate-parameter.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is a feature, not a bugfix:
```suggestion
minor_changes:
```